AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/justice-user-form/justice-user-form.component.ts

```diff
@@ -50,7 +53,10 @@ export class JusticeUserFormComponent implements OnChanges {
             contactTelephone: new FormControl('', [Validators.pattern(Constants.PhonePattern)]),
             firstName: new FormControl('', [Validators.pattern(Constants.TextInputPatternName)]),
             lastName: new FormControl('', [Validators.pattern(Constants.TextInputPatternName)]),
-            role: new FormControl(this.availableRoles.Vho)
+            roles: new FormControl([], [justiceUserRoleValidator()]),
+            rolevho: new FormControl(false),
```

Comment:
you only need roles and not the role specific one

Reply:
sorry but I need this as they are in interface and it does not let me remove them in the constructor

Reply:
Resolving in place of the form array conversation